<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWidget</class>
 <widget class="QWidget" name="tabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="zaglavlje" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="title">
        <property name="styleSheet">
         <string notr="true">QLabel { 
font: 75 italic 10pt &quot;Droid Serif&quot;; color: rgb(234, 234, 234)}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;N A S L O V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="controls" native="true">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{background-color: rgb(177, 177, 177);
border-style: inset;
border-top-right-radius: 8px;}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="minimize">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(255, 213, 0);
border-radius: 5px;
	font: 75 15pt &quot;Comic Sans MS&quot;;
font-weight: bold;
}
QPushButton:hover{background-color: white}</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maximize">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(255, 213, 0);
border-radius: 5px;
	font: 75 15pt &quot;Comic Sans MS&quot;;
font-weight: bold;
}
QPushButton:hover{background-color: white}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.0511364 rgba(219, 206, 12, 239), stop:0.8125 rgba(250, 234, 5, 255), stop:1 rgba(86, 144, 148, 0));
border-radius: 5px;
	font: 75 10pt &quot;Comic Sans MS&quot;;
font-weight: bold;
}
QPushButton:hover{background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(148, 1, 1, 255), stop:0.505682 rgba(214, 5, 5, 228), stop:0.505682 rgba(214, 84, 84, 255), stop:1 rgba(86, 144, 148, 0))}</string>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="slotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{background-color: qlineargradient(spread:pad, x1:0.749228, y1:0.108, x2:0.119318, y2:0.909091, stop:0 rgba(177, 177, 177, 245), stop:1 rgba(247, 247, 247, 255));
border-style: inset;
border-top-left-radius: 8px;
border-bottom-left-radius: 8px;
border-bottom-right-radius: 8px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlWidgetDummy" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="control2Widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="controlWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
